speed-dreams/source-2.2.3/src/drivers/assistedhuman/Driver.cpp

```diff
@@ -38,7 +41,18 @@ void Driver::InitTrack(tTrack* p_track, void* p_carHandle, void** p_carParmHandl
     m_humanDriver.init_track(m_index, p_track, p_carHandle, p_carParmHandle, p_situation);
     m_humanDriver.SetRecorder(useRecorder ? m_recorder : nullptr);
 
+    auto start = std::chrono::system_clock::now();
     SMediator::GetInstance()->RaceStart(p_track, p_carHandle, p_carParmHandle, p_situation, m_recorder);
+    auto end = std::chrono::system_clock::now();
+
+    long milliseconds = static_cast<long>(std::chrono::duration_cast<std::chrono::milliseconds>(end - start).count() / BLACK_BOX_TESTS);
```

Comment:
Add comment explaining why you divide by BLACK_BOX_TESTS (I know why but future devs won't)